docs/src/lib.rs

```diff
@@ -245,23 +247,27 @@ fn category_page(resolver: &dyn Resolver, category: Category) -> PageModel {
         shorthands = Some(ShorthandsModel { markup, math });
     }
 
+    let mut skip = HashSet::new();
+    if category == MATH {
+        // Already documented in the text category.
+        skip.insert("text");
```

Comment:
This is no-op becase `skip` is overwritten on the next line

Reply:
dumb me

Reply:
Will address in https://github.com/typst/typst/pull/5089